### DISCUSSION TITLE
MS Defender: Multi-stage incident involving Defense evasion & Discovery on one endpoint

### DISCUSSION THREAD

**Discussion by netSpeedyTV** [rustdesk/rustdesk-server-pro#308] [General]
We have also been using the Pro version since yesterday and have now created a custom client. This was immediately classified as dangerous by MS Defender. 

I assume that other virus scanners will also recognize this. What can we do so that we don't have this problem? 

Thank you.

**Comment by rustdesk** [comment]
Is it signed? sometimes, signing custom client may fail, if you see not signed, please build a new custom client.

Actually our public release is also recognized by some virus scanners, but not by defender so far.

**Comment by netSpeedyTV** [comment]
Are there instructions on how I should proceed there? To sign the client. Then does it need SSL? This is the next step I'm trying to use Traefik from another environment to run the domain:21114 over it. Unfortunately it doesn't work at the moment.

**Reply by rustdesk** [reply]
We sign it when building custom client, but it may fail, please build again with web console if you see it not signed.

**Reply by rustdesk** [reply]
About ssl, https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/#setup-https-for-web-console
